src/layouts/AccessWalletLayout/components/HardwareModal/HardwareModal.vue

```diff
@@ -166,6 +171,15 @@ export default {
           disabled: false,
           msg: '',
           link: 'http://lddy.no/a4im'
+        },
+        {
+          name: COOLWALLET_TYPE,
+          imgPath: coolwallet,
+          text: 'CoolWallet',
+          disabled: false,
+          msg: '',
+          link:
+            'https://coolwallet.io/product/coolwallet/?ref=myetherwalletmyetherwallet'
```

Comment:
ref= has myetherwalletmyetherwallet is this right?

Reply:
I got this from them so im not too sure. I'll ask